### ISSUE TITLE
Not installing in Google Colab 

### ISSUE THREAD

**Issue by goncamateus** [PeterRochford/SkillMetrics#20]
Hello guys!
I'm trying to install the package in the Google Colab environment but I keep receiving this:
```
Collecting SkillMetrics
  Downloading https://files.pythonhosted.org/packages/6f/f0/0365c196295c6f4356298b1369c9121c71d90b52755807751d00d2a6fff6/SkillMetrics-1.1.7.tar.gz (44kB)
     |████████████████████████████████| 51kB 2.6MB/s 
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```
Does anyone know why?
Thank you in advance!

**Comment by PeterRochford** [comment]
I do not use the Google Colab environment so I don't have any constructive suggestions. 

Note that a new version of SkillMetrics is now available: 1.2.1. You may wish to try installing this latest version to see if it works for you.

**Comment by PeterRochford** [comment]
Other users have posted a solution that appears to work for Google Colab. I've added it to the FAQ under [Installation Issues](https://github.com/PeterRochford/SkillMetrics/wiki/FAQ#installation-issues) as Q2. I hope this helps.
